data/fixtures/recorded/languages/clojure/clearItemFine.yml

```diff
@@ -21,7 +21,7 @@ initialState:
       start: {line: 0, character: 2}
       end: {line: 0, character: 5}
 finalState:
-  documentContents: "{ :baz \"whatever\"}"
+  documentContents: "{}"
```

Comment:
NTS: this outcome is worse but there's not much we can do about it at the current time. Problem is that closure does not use explicit list limiters and the textual base system can obviously not figure out if a space is part of an item or a separator here. The textual system thinks the entire `:foo "bar" :baz "whatever"` is one "item".

Andreas' idea: allow a language to specify whether it uses explicit delimiters for lists, and if it does not, go back to the old implementation of always taking the language servers opinion.

Reply:
https://github.com/cursorless-dev/cursorless/issues/2723